server/views/pages/activities/allocation-dashboard/activities.njk

```diff
@@ -87,7 +87,7 @@
                     {
                         attributes: {
                             id: 'activity-' + loop.index,
-                            "data-sort-value": activity.activityName
+                            "data-sort-value": activity.activityName | upper
```

Comment:
I used the nunjucks upper method to uppercase the data just on the sort values. The `fix-rows.js` is a function that runs on the line 32 for the activities row via its 'data-sort-fixed' attribute. The function users greater than and therefore appears to sort by ASCII standards, overriding the standard GDS component sort behaviour